### PR TITLE
Update db basics and advanced.

### PR DESCRIPTION
**Pull request by qbart**
Finalized verison of basic/adv bubble.
Open for feedback.

### CODE REVIEW

**Comment by qbart** [comment]
I've added @wojtodzio for review as well.

File: content/web_development/skills/persistence/advanced.md [hunk 1]
```diff
@@ -1,36 +1,38 @@
...
-**Description:** You can get the most out of the relational database (PostgreSQL) and knows when to pick different solution for persistance layer
+**Description:** You can get the most out of the relational database (PostgreSQL) and know when to pick different solution for the persistence layer
 
-**Person which successfully completed requirement for given block can:** 
+**A person who successfully completed requirement for the given block:** 
 
-- Knows other type of databases and common use-cases for them
-- Can build reactable database solution
+- Knows other types of databases and common use-cases for them
+- Can build a reactable database solution
```
Comment (wojtodzio) [inline content/web_development/skills/persistence/advanced.md:17]: I'm not sure what does "reactable" database solution means, isn't it a typo?

**Review by wojtodzio — APPROVED** [review]
Awesome job!
I've found a few tiny typos and grammar mistakes, most of which were outside of the original commit, so I decided that it would be easier for me to push a commit with fixes instead of commenting.
My commit made the diff way longer since I replaced "Person which" with "Person who" project-wise and renamed a few files - sorry about that! I know that was outside of the scope of this PR, but I just couldn't help myself ;)

File: content/web_development/skills/persistence/advanced.md (hunk 1 above)
Comment (qbart) [reply content/web_development/skills/persistence/advanced.md:17]: Thanks, even small contributions count :1st_place_medal: 
When it comes to a "reactable" I don't know either - I'll change it to sth similiar

**Comment by PoslinskiNet** [comment]
@qbart looks great 👍🏼

**Pull request merged**